app/controllers/api/subcollections/alert_definitions.rb

```diff
@@ -0,0 +1,10 @@
+module Api
+  module Subcollections
+    module AlertDefinitions
+      def alert_definitions_query_resource(object)
+        return {} unless object.respond_to?(:miq_alerts)
+        object.miq_alerts
```

Comment:
you mean `MiqAlertSet` and `MiqAlert` right?

Are you sure about it? seemed to me it returns the object's `miq_alerts`, regarding of the set type.

BTW, to the best of my knowledge, there is no `miq_alert_mixin.rb` to offer get_alerts